common/main/km.xml

```diff
@@ -4640,17 +4640,6 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<symbols numberSystem="khmr">
 			<decimal>,</decimal>
 			<group>.</group>
-			<list draft="contributed">↑↑↑</list>
```

Comment:
So I didn't vote for explicit inheritance here. Should we revert the km import, or should I go in and vote for explicit inheritance?

Reply:
That i'm not sure, but i think for now we should just fix it in XML if needed?

Reply:
Wouldn't it just automatically inherit the default numbering system formatting even if it's not explicitly present?

Reply:
i'd think so. it'll just be missing next time.